public/typescript/__test__/RegistrationPageService.test.ts

```diff
@@ -126,25 +125,20 @@ describe('RegistrationPageService', () => {
       if (!statusCallback || !errorCallback) {
         throw new Error('Should have started status request');
       }
-      window.document.location.replace = jest.fn();
       statusCallback(RegistrationStateMachine.FINALIZED);
     });
 
-    it.skip('SKIPPED TEMPORARILY! The qr code should be hidden', () => {
+    it('The qr code should be hidden', () => {
       expect(context.qrComponent.isVisible()).toBeFalsy();
     });
 
-    it.skip('SKIPPED TEMPORARILY! Polling should be disabled', () => {
+    it('Polling should be disabled', () => {
       expect(context.pollingService.enabled).toBeFalsy();
     });
 
-    it.skip('SKIPPED TEMPORARILY! Show finalized', () => {
+    it('Show finalized', () => {
       expect(context.statusUi.showFinalized).toBeCalled();
     });
-
-    it.skip('SKIPPED TEMPORARILY! Redirect to finalized page', () => {
-      expect(document.location.replace).toBeCalledWith('http://fake-finalized-url.com');
-    });
```

Comment:
Testing dom interactions should not be part of Jest tests without having a robust browser mock in place. It was not feasible to include that in this PR.